### ISSUE TITLE
Allow rerendering TUI on the `SIGHUP` signal

### ISSUE THREAD

**Issue by ringeringeraja** [charmbracelet/glow#524]
This way someone could watch for file changes using a tool like entr and get a live preview while editing a Markdown file.

**Comment by brianstrauch** [comment]
It looks like `find README.md | entr -cs "glow README.md"` could accomplish this. Did you have something else in mind @minenwerfer?

**Comment by minenwerfer** [comment]
> It looks like `find README.md | entr -cs "glow README.md"` could accomplish this. Did you have something else in mind @minenwerfer?

Yeah, I thought of just rerendering the document without missing the cursor point in "pager" mode to allow live preview while editing. Mupdf does it this way (rerendering seamlessly on SIGHUP), so entr can be used to send the signal on file changes.

The command you've sent does the trick for smaller documents, though.

**Comment by brianstrauch** [comment]
Sounds like the goal would be to get the following to work?
```
glow README.md --pager & ls README.md | entr /bin/sh -c "pkill -HUP glow"
```

**Comment by minenwerfer** [comment]
> Sounds like the goal would be to get the following to work?
> 
> ```
> glow README.md --pager & ls README.md | entr /bin/sh -c "pkill -HUP glow"
> ```

Exactly.
